Dockerfile

```diff
@@ -9,7 +9,6 @@ FROM docker.io/library/golang:1.16.2 as builder
 RUN echo 'deb http://deb.debian.org/debian buster-backports main' >> /etc/apt/sources.list
 
 RUN apt-get update && apt-get install -y \
-    openssh-server \
```

Comment:
The openssh-server is required for e2e tests. Does it have to be removed ?

Reply:
reverted openssh-server removal. @alexmt